LayoutTests/storage/indexeddb/cursor-continue-add-failure-private.html

```diff
@@ -0,0 +1,10 @@
+<!-- webkit-test-runner [ useEphemeralSession=true ] -->
```

Comment:
I'm curious, what's the difference between this test and the other one?

Reply:
This means to use ephemeral session (private browsing) for this test. 
We have two different backends (memory backend & disk backend), and we want to test this case with both of them.

Reply:
That makes sense. Thank you for clarifying.